js/app.js

```diff
@@ -0,0 +1,170 @@
+function displayQuestions() {
+    $('#userChoices').empty();
+    //Display question text
+    $('#addQuestionText').text(questions[questionNum].question);
+    //display image
+    $('#image').html('<img src =" ' + questions[questionNum].imageUrl + '" class="image-class" />');
+    //show choices
+    var totalChoices = questions[questionNum].choices.length;
+    for (var i = 0; i < totalChoices; i++) {
+        $('#userChoices').append("<li>" + "<input id='option' class='option' type='radio' name='option' value='" + i + "'>" + (questions[questionNum].choices[i] + "</li>"))
+    } //update question tracker
+    $('#question-counter').text("Question " + (questionNum + 1) + "/" + questions.length);
+    //update score tracker
+    $('#score').text('Score ' + (numCorrect) + "/" + questions.length);
+
+    var htmlOutput = "";
+    htmlOutput += '<div class="answers">';
+    htmlOutput += '<div class="answer-text"> <h3>' + (questionNum + 1) + ') ' + 'Q: ' + questions[questionNum].question + '</h3>';
+    htmlOutput += '<p>A: ' + questions[questionNum].details + '</p></div>';
+    htmlOutput += '<div class="answer-image"><img src ="' + questions[questionNum].imageUrl + '" class="image-class" /></div>';
+    htmlOutput += '</div>';
+    $('.answers-container').append(htmlOutput);
+}
+
+//loading questions
+function nextQuestion() {
+    //update score
+    var userGuess = $("input[id='option']:checked").val();
+    var correctAnswer = questions[questionNum].correct;
+    //console.log("userGuess: " + userGuess);
+    //console.log("correctAnswer: " + correctAnswer);
+    if (userGuess == correctAnswer) {
+        numCorrect++;
+        console.log(numCorrect);
+        //alert('I have activated the function nextQuestion')
+    }
+    //submit final question
+    if (questionNum + 1 == questions.length) {
+        //update final score at top
+        $('#finalScore').text("You got " + numCorrect + '/' + questions.length + ' right!')
+            //show final section only
+        $('.questions').hide();
+        $('.introduction').hide();
+        $('.final').show();
+    } //load another question
+    else {
+        questionNum++;
+        displayQuestions();
+    }
+}
+
+
+//try again
+function tryAgain() {
+    document.location.reload(true);
+    //alert('I have activated the tryAgain function')
+}
+
+//Step 3 Use Functions
+$(document).ready(function () {
+    //on page load
+$('.questions').hide();
+$('.final').hide();
+$('.introduction').show();
+
+//starts quiz
+$('#start').on('click', function () {
+    $('.introduction').hide();
+    $('.final').hide();
+    $(".questions").show();
+    displayQuestions();
+})
+
+    //hits guess button
+    $('#submit').on('click', function () {
+        if ($("#userChoices input[id='option']").is(':checked')) {
+            nextQuestion();
+
+        } else {
+            alert("Please choose an option!");
+        }
+    })
+
+    //hits try again button
+    $('#reset').on('click', function () {
```

Comment:
instead of an anonymous function that triggers a named function, you can trigger the named function by passing as an argument: 
$('#reset').on('click',tryAgain)